src/Components/Buttons/button.css

```diff
@@ -0,0 +1,92 @@
+/** @format */
+
+:root {
+	--primary: #4628ff;
+	--warning: #ffd028;
+	--danger: #eb3f27;
+	--success: #13df27;
+	--white: #fdfdfd;
+	--dark: #181717;
+}
+
+.btn {
+	font-family: "Roboto", sans-serif;
```

Comment:
Define this as a variable in the common variables file and access it as a variable.

Reply:
Resolved